### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Mre55**
### In this PR:

- Created a `main.rb` file that will serve as console app entry-point. 
- In the `main.rb` define the entry point. This method do the following:

      > Present the user with a list of options to perform.
      
      > Lets users choose an option.
      
      > Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  end
+
+  def do_action(option)
+    case option
+    when '1'
+      @actions.list_books
+    when '2'
+      @actions.list_people
+    when '3'
+      @actions.create_person
```
Comment (omar-labana) [inline main.rb:19]: - [x] Creating a teacher prompts for `parent permission and `specialization` where it should ask only for the latter, please update it.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  def init
+    main while @actions.keep_going
+  end
+
+  def do_action(option)
+    case option
+    when '1'
+      @actions.list_books
+    when '2'
+      @actions.list_people
```
Comment (omar-labana) [inline main.rb:17]: - [x] Listing people shows unwanted repeating "name: Unkown, id, age" before each person data, kindly fix it so it matches the requirements.
- [x] There should be no parent permission in any of the teacher's entries, instead, it should be the teacher's specialization.
image

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    when '3'
+      @actions.create_person
+    when '4'
+      @actions.create_book
+    when '5'
+      @actions.create_rental
+    when '6'
+      @actions.list_rentals
```
Comment (omar-labana) [inline main.rb:25]: - [x] After entering a person's id to list its rentals the app crashes with this error "Ruby-OOP-school-library/school/actions/actions.rb:100:in `list_rentals': undefined method `filter' for #<Array:0x0000556af41fd8d0> (NoMethodError)".

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Mre55, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
RAEDME is neatly formatted and linters are passing. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Mre55, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Great job working on the required changes 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**